tests/CMakeLists.txt

```diff
@@ -220,13 +220,20 @@ endif()
 # Bash tests
 
 if(AVIF_BUILD_APPS)
+    add_test(NAME test_cmd COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd.sh ${CMAKE_BINARY_DIR}
+                                   ${CMAKE_CURRENT_SOURCE_DIR}/data
+    )
     # When building apps, test the avifenc/avifdec.
     # 'are_images_equal' is used to make sure inputs/outputs are unchanged.
     add_executable(are_images_equal gtest/are_images_equal.cc)
     target_link_libraries(are_images_equal aviftest_helpers)
-    add_test(NAME test_cmd COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/test_cmd.sh ${CMAKE_BINARY_DIR}
-                                   ${CMAKE_CURRENT_SOURCE_DIR}/data
-    )
+    if(MINGW)
+        if("$ENV{MSYSTEM}" MATCHES "(MINGW32|MINGW64)")
+            set_tests_properties(test_cmd PROPERTIES ENVIRONMENT "AVIF_TEST_NO_UTF8=1")
```

Comment:
Nit: Many people recommend against defining a boolean flag in the negative sense. To follow that recommendation, we will need define the `AVIF_TEST_UTF8` boolean flag, set it to 1 by default, and set it to 0 here.